### PR TITLE
feat: add config to control built php versions

### PR DESCRIPTION
**Pull request by shouze**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 SHELL := /bin/bash
-php_versions ?= 73 74
 layer ?= *
 parallel = $(if $(shell which parallel),true,false)
+resolve_php_versions = $(or $(php_versions),`jq -r '.php | join(" ")' ${1}/config.json`)
```
Comment (shouze) [inline Makefile:4]: imply to have jq installed, should I change of something else? I can try doing it in php.
should look like that:
```sh
php -r 'echo join(" ", json_decode(file_get_contents("${1}/config.json"), true)["php"]);'
```

Comment (Nyholm) [reply Makefile:4]: Yeah, But that imply that you have PHP installed =)

As long as it works on github actions then Im fine either way.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,12 @@
 # Change log
 
+## 0.7.0
```
Comment (Nyholm) [inline CHANGELOG.md:3]: Lets
```suggestion
## 0.6.4
```

File: Makefile (hunk 1 above)
Comment (Nyholm) [reply Makefile:4]: Nice with `or` btw.

File: layers/amqp/Dockerfile [hunk 3]
```diff
@@ -1,13 +1,13 @@
 ARG PHP_VERSION
-FROM bref/build-php-$PHP_VERSION:1.0.0-beta1 AS ext
+FROM bref/build-php-$PHP_VERSION:1.0.0 AS ext
```
Comment (Nyholm) [inline layers/amqp/Dockerfile:2]: Good!

**Review by Nyholm — APPROVED** [review]
Excellent work, thank you

File: Makefile (hunk 1 above)
Comment (shouze) [reply Makefile:4]: Yes, this is fully BC and very handy when you just want to rebuild some specific layer variant.

**Comment by Nyholm** [comment]
I’ll merge this. Let’s see if it builds properly on master 👍 

Rock n’ Roll :)

**Pull request merged**